public/resources/js/form.js

```diff
@@ -1,76 +1,89 @@
-    var pass = document.getElementById("password");
-    var msg = document.getElementById("message");
-    var str = document.getElementById("strenght");
+var icon = document.getElementsByClassName("icon");
+var pass = document.getElementById("password");
+var msg = document.getElementById("message");
+var str = document.getElementById("strenght");
 
-    var copass = document.getElementById("cpassword");
-    var nmsg = document.getElementById("nomessage");
-    var nostr = document.getElementById("constrength");
+var copass = document.getElementById("cpassword");
+var nmsg = document.getElementById("nomessage");
+var nostr = document.getElementById("constrength");
 
-    var email = document.getElementById("email");
-    var emsg = document.getElementById("emailmsg");
-    var estr = document.getElementById("emailstrenght");
+var email = document.getElementById("email");
+var emsg = document.getElementById("emailmsg");
+var estr = document.getElementById("emailstrenght");

```

Comment:
this is the first error i was talking about in the other comment, the naming is to general, if you have something like an obj
register = {
  icons: document.getElementsByClassName("icon");
}
it will be more intuitive for what you are talking about: register.icons
even you are not into that way of code, you should improve the way you name your variables, like nostr, estr, emsg, just you will understands the concepts because you code but any other teammate will have a bad time figuring out for what you are using this elements. Leaving that, we have jsquery so you could avoid the work of write the document.get-something thing and instead just call $('') to a better rediability